packages/dashboard/src/redux/selectors.ts

```diff
@@ -14,6 +22,13 @@ const EMPTY_ARRAY = Object.freeze([]);
 
 type Selector<R> = (state: RootState) => R;
 
+// https://react-redux.js.org/using-react-redux/usage-with-typescript#define-typed-hooks
+// Defined in @deephaven/dashboard, as that's the most common package with React and @deephaven/redux as dependencies
+// We could have another package specifically for this, but it's not really necessary.
+export const useAppDispatch: () => RootDispatch = useDispatch;
```

Comment:
Once this merges, would probably be good to announce the new API to let people know